feat(curriculum): add steps for employee profile generator workshop (#64896)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/blocks/workshop-employee-profile-generator/694bf6ea530e19a9c48c59f0.md

---
id: 694bf6ea530e19a9c48c59f0
title: Step 2
challengeType: 20
dashedName: step-2
---

# --description--

If you simply add two names like `"John" + "Doe"`, the result is `"JohnDoe"` with no space.
To fix this, you need to concatenate a string containing a space `" "` between them.

Update your `full_name` variable so it concatenates `first_name`, a space `" "`, and `last_name`.

# --hints--

You should add a space `" "` between `first_name` and `last_name`.

```js
({ test: () => true })
```

# --seed--

## --seed-contents--

```python
first_name = "John"
last_name = "Doe"

--fcc-editable-region--

--fcc-editable-region--

> ```

Reviewer: missing `full_name` variable

---
id: 694bf6ea530e19a9c48c59f0
title: Step 3
challengeType: 20
dashedName: step-3
---

# --description--

If you simply add two names like `'John' + 'Doe'`, the result is `'JohnDoe'` with no space.
To fix this, you need to concatenate a string containing a space `' '` between them.

> Update your `full_name` variable so it concatenates `first_name`, a space `' '`, and `last_name`.

Reviewer 2: ```suggestion
Update your `full_name` variable so it concatenates `first_name`, a space, and `last_name`.
```

# --hints--

You should add a space `' '` between `first_name` and `last_name`.

```js
({ test: () => true })
```

# --seed--

## --seed-contents--

```python

--fcc-editable-region--
first_name = 'John'
last_name = 'Doe'
full_name = first_name + last_name

> 

Reviewer 2: Missing `print`s

---
id: 694bf6ea530e19a9c48c59f0
title: Step 3
challengeType: 20
dashedName: step-3
---

# --description--

> If you simply add two names like `'John' + 'Doe'`, the result is `'JohnDoe'` with no space.

Reviewer 2: ```suggestion
If you concatenate two strings like `'John' + 'Doe'`, the result is `'JohnDoe'` with no space.
```

To fix this, you need to concatenate a string containing a space (`' '`) between them.

Update your `full_name` variable so it concatenates `first_name`, a space, and `last_name`.

# --hints--

You should add a space (`' '`) between `first_name` and `last_name`.

```js
({ test: () => true })
```

# --seed--

## --seed-contents--

```python

--fcc-editable-region--
first_name = 'John'
last_name = 'Doe'
print(first_name)
print(last_name)
full_name = first_name + last_name

> --fcc-editable-region--

Reviewer 2: ```suggestion
first_name = 'John'
last_name = 'Doe'
print(first_name)
print(last_name)
--fcc-editable-region--
full_name = first_name + last_name
--fcc-editable-region--
print(full_name)
```

```python
first_name = "John"
last_name = "Doe"

> full_name = first_name + last_name

Reviewer: you are asking to change this, so it should be inside the editable region
